### ISSUE TITLE
Fail when using the stabilized feature `weak-dep-features`

### ISSUE THREAD

**Issue by jcornaz** [est31/cargo-udeps#116]
the `weak-dep-feature` has been stablized in rust 1.60. In a project that uses it, cargo-udeps fails: 

```sh
cargo +nightly -Zweak-dep-features udeps --workspace --all-features
```

```
warning: flag `-Z weak-dep-features` has been stabilized in the 1.60 release, and is no longer necessary
  Weak dependency features are now always available.

error: failed to load manifest for workspace member `/home/jonathan/Desktop/heron/debug`

Caused by:
  failed to load manifest for dependency `heron_rapier`

Caused by:
  failed to parse manifest at `/home/jonathan/Desktop/heron/rapier/Cargo.toml`

Caused by:
  optional dependency features with `?` syntax are only allowed on the nightly channel and requires the `-Z weak-dep-features` flag on the command line
  Feature `enhanced-determinism` had feature value `rapier2d?/enhanced-determinism`.
```

Note: cargo check passes, using either stable or nightly channel.

**Comment by est31** [comment]
Yeah there needs to be an upgrade to the newer cargo version. Great to hear that this has finally become stable. PRs are welcome, otherwise I might do it later this week.

**Comment by jcornaz** [comment]
> Can you try with latest master if the issue persists?

Seems to work

**Comment by est31** [comment]
Great! Fixed by 65e1d53a796fe356dd639a56055ca872b2d33f18 then :)
